tests/lib/ocpp/v201/test_profile.cpp

```diff
@@ -26,11 +26,14 @@ using std::nullopt;
 using std::chrono::minutes;
 using std::chrono::seconds;
 
+auto FIXME_DEFAULT_LIMIT = 0;
```

Comment:
```suggestion
// it was required to introduce this default because the `limit` property became optional in the 2.1 message
auto FIXME_DEFAULT_LIMIT = 0;
```